chrome/content/zotero/zoteroPane.js

```diff
@@ -4550,6 +4552,28 @@ var ZoteroPane = new function()
 			});
 		}
 	});
+
+	this.createParentItemFromIdentifier = async function (node) {
+		if (!this.canEdit()) {
+			this.displayCannotEditLibraryMessage();
+			return;
+		}
+
+		let items = this.getSelectedItems();
+		for (let item of items) {
+			if (!item.isTopLevelItem() || item.isRegularItem()) {
+				throw new Error('Item ' + item.itemID + ' is not a top-level attachment or note in ZoteroPane_Local.createParentItemFromIdentifier()');
+			}
+
+			Zotero_Lookup.showPanelXY(node.popupBoxObject.x, node.popupBoxObject.y, {
+				libraryID: item.libraryID,
+				collections: item.getCollections(),
+				id: item.id
```

Comment:
Can't we just pass `item` here as `childItem`, store the actual item, and get the libraryID and collections from that in Zotero_Lookup?